tasks/options/karma.js

```diff
@@ -3,15 +3,15 @@ module.exports = {
         configFile: 'karma.conf.js',
         singleRun: true,
         autoWatch: false,
-        reporters: ['dots'],
+        reporters: ['spec'],
     },
     single: {
-        reporters: 'dots',
+        reporters: 'spec',
     },
     watch: {
         singleRun: false,
         autoWatch: true,
-        reporters: 'dots',
+        reporters: ['progress'],
```

Comment:
Is this correct, previously this was a string and now an array?

Reply:
I changed it for testing, will set it back to a string.